pre_commit_hooks/requirements_txt_fixer.py

```diff
@@ -69,10 +69,11 @@ def fix_requirements(f):
     else:
         rest = []
 
+    # find and remove pkg-resources==0.0.0
+    # which is automatically added by broken pip package under Debian
     for requirement in requirements:
-        if b'pkg-resources' in requirement.name:
-            if b'0.0.0' in requirement.value:
-                requirements.remove(requirement)
+        if requirement.value == b'pkg-resources==0.0.0\n':
+            requirements.remove(requirement)
```

Comment:
actually, now that I look at it I think this loop results in UB (modifying list while iterating over it) -- you can avoid this with:

```python
requirements = [
    requirement for requirement in requirements
    if requirement.value != b'pkg-resources==0.0.0\n'
]
```